### PR TITLE
feat(apm): added apm configuration setup

### PR DESCRIPTION
**Pull request by michaelmass**
This adds the Sentry sdk inside the messaging server to monitor application metrics (apm).

### CODE REVIEW

**Comment by samuelmasse** [comment]
What does this do? Can you add a description to you PR. There are prettier errors in the ordering of imports. Please make sure to install the prettier extension for vscode before committing to this repo. The name of the PR should also follow standard commit norms. So : chore(server): add apm

File: packages/server/src/api.ts [hunk 1]
```diff
@@ -9,6 +9,8 @@ import { MessageApi } from './messages/api'
 import { SocketManager } from './socket/manager'
 import { SyncApi } from './sync/api'
 import { UserApi } from './users/api'
+import * as Sentry from '@sentry/node'
```
Comment (laurentlp) [inline packages/server/src/api.ts:12]: Make sure that ESLint and Prettier run on your code. This should be automatic when using VSCode.

**Comment by samuelmasse** [comment]
@michaelmass can you fix these issues so we can merge this?

**Review by laurentlp — APPROVED** [review]
LGTM!

**Review by samuelmasse — APPROVED** [review]

**Pull request merged**